src/loaders/express.js

```diff
@@ -76,9 +76,23 @@ export default (app) => {
     }
 
     // Adding headers (CORS)
-    app.use((_, res, next) => {
-        // Allow connections for all origins
-        res.setHeader("Access-Control-Allow-Origin", config.access_control_allow_origin);
+    app.use((req, res, next) => {
+        // Allow connections for default specified connections
+        res.setHeader("Access-Control-Allow-Origin", "null");
+
+        // Allow requests from connections specified by the allow list regexes
+        const origin = req.header("origin")?.toLowerCase();
```

Comment:
Why convert to lower case? Might cause some misunderstandings with the regex provided in the config file